From 8f08d48405c689cea5d91fe9e9d77b09c7b0c378 Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Wed, 11 Sep 2019 11:51:47 +0200
Subject: [PATCH] Remove 'phonon' from the include directory

Summary:
Both the CMake and pkgconfig files say the include directory is
`<prefix>/include/phonon4qt5`. Only the .pri file points to a different location.

This fixes the nulloy build with recent phonon versions.

Reviewers: sitter, O4 phonon

Differential Revision: https://phabricator.kde.org/D23873
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 85290887..ccf65c68 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -170,7 +170,7 @@ ecm_generate_pri_file(
    LIB_NAME ${PHONON_LIB_SONAME}
    DEPS "core widgets"
    FILENAME_VAR pri_filename
-   INCLUDE_INSTALL_DIR ${INCLUDE_INSTALL_DIR}/phonon
+   INCLUDE_INSTALL_DIR ${INCLUDE_INSTALL_DIR}
 )
 install(FILES ${pri_filename} DESTINATION ${ECM_MKSPECS_INSTALL_DIR})
 
-- 
2.23.0

